mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -85,6 +85,11 @@ CLASS net/minecraft/unmapped/C_qceyzrkd net/minecraft/block/Blocks
 		ARG 2 pos
 	METHOD m_suyzsusa (Lnet/minecraft/unmapped/C_txtbiemp;)I
 		ARG 0 state
+	METHOD m_sxxhsshv createLogBlock (Lnet/minecraft/unmapped/C_ifcgjcqy;Lnet/minecraft/unmapped/C_ifcgjcqy;Lnet/minecraft/unmapped/C_aevintex;[Lnet/minecraft/unmapped/C_kksdgidr;)Lnet/minecraft/unmapped/C_djwplmtn;
+		ARG 0 uprightColour
+		ARG 1 sidewaysColour
```

Comment:
```suggestion
		ARG 0 uprightColor
		ARG 1 sidewaysColor
```
Use US English spellings for consistency, please!

Reply:
done!